### ISSUE TITLE
Add comman line

### ISSUE THREAD

**Issue by h2non** [h2non/filetype.py#2]

**Comment by vuolter** [comment]
Usage of thirty modules like [click ](http://click.pocoo.org/) is allowed, right?!

**Comment by h2non** [comment]
Yeah, click looks neat, but for what we need here `argparse` is enough and built-in the stdlib.

**Comment by E-RROR** [comment]
Can I Handle It ?

**Comment by h2non** [comment]
PR are very welcome!

**Comment by dosas** [comment]
https://github.com/h2non/filetype.py/pull/98

**Comment by codyswanner** [comment]
Hi, just found this and it looks very useful for something I'm working on, I'm interested in seeing if I can help out in any way!  This issue links a couple merged pull requests, has this already been done?
